include/sys/fs/zfs.h

```diff
@@ -1123,6 +1124,13 @@ typedef enum {
 	VDEV_INITIALIZE_COMPLETE
 } vdev_initializing_state_t;
 
+/*
+ * nvlist name constants. Facilitate restricting snapshot iteration range for
+ * the "list next snapshot" ioctl
+ */
+#define	SNAP_ITER_SKIP_AFTER_TXG	"snap_iter_skip_after_txg"
+#define	SNAP_ITER_SKIP_TO_TXG		"snap_iter_skip_to_txg"
```

Comment:
how about `SKIP_AFTER_TXG` and `SKIP_BEFORE_TXG`, or perhaps better yet,`SNAP_ITER_MAX_TXG` and `SNAP_ITER_MIN_TXG`?